pages/ag/neuro-rse.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: Neuro-RSE Affinity Group
+subtitle:
+menubar:
+permalink: /ag/neuro-rse/
+set_last_modified: true
+---
+# Overview
+
+Neuroscience RSE (Neuro-RSE) is an affinity group for those working on research software in neuroscience. We meet to support each other and discuss common issues and concerns.
+
+# Membership
+
+Neuro-RSE is open to anyone who considers themselves a research software engineer and who work in the field of neuroscience, regardless of job title or domain. This includes grad students and postdocs, as well as those in permanent positions with "engineer" or "scientist" in their title.
+
+Membership is open to all [US-RSE members](https://us-rse.org/join/). As with other US-RSE activities, Neuro-RSE is US-focused, but other countries are welcome.
+
+To join, contact one of the coordinators on the US-RSE slack (see the [Questions?](#questions) section below).
+
+# Activities
+
+Neuro-RSE meets virtually monthly, with alternating times (one more East Coast-friendly, one more West Coast-friendly). Additional in-person meetings at RSE or Neuroscience events may also be organized.
+
+During our meetings, we have brief lightning talks, to share our work with each other, as well as discussions on relevant topics. These topics are determined by group members and may include: how are RSEs managed and incentivized in neuroscience, how to convince scientists of the value of our work (and the time required), what does day-to-day work look like, how to incentivize building / using common resources instead of reinventing the wheel, how to formally run teams and open source collaborations.
+
+We have a Slack channel in the US-RSE space for discussions.
+
+# Questions?
+
+Reach out to the Neuro-RSE coordinators Billy Broderick or David Nicholson on the [US-RSE slack](https://us-rse.org/join/) with questions.
+
+# Active members
+
+Active members attend our regular meetings at least once a quarter (once every 3 meetings).
+
+- [Billy Broderick](wfbroderick.com/), data scientist at [Flatiron Institute's Center For Computational Neuroscience](https://www.simonsfoundation.org/flatiron/center-for-computational-neuroscience/) 
```

Comment:
IMHO: might be worth sorting as well. If there is an official lead, best to make leadership position explicit/separate